src/frame/js/smooch.jsx

```diff
@@ -209,13 +220,9 @@ export function login(userId = '', jwt, attributes) {
             id: loginResponse.appUser.userId || loginResponse.appUser._id
         });
 
-        Raven.setExtraContext({
-            appToken
```

Comment:
yeah it's done here https://github.com/smooch/smooch-js/pull/572/files#diff-8f16faf8c766689cc06ac2f62eb9db67R129